code/_helpers/unsorted.dm

```diff
@@ -1159,6 +1159,9 @@ var/list/wall_items = typecacheof(list(
 			to_chat(user, "<span class='notice'>You need to be holding [src] to do that.</span>")
 		return USE_FAIL_NOT_IN_USER
 
+/atom/proc/use_check_and_message(mob/user, use_flags = 0, show_messages = TRUE)
+	..()

```

Comment:
...I am surprised that this even compiles. This returns no value and it does no action. It should be:
(And you can remove the argument from this proc, since using this proc with the argument set to `FALSE` makes no sense.)
```dm
/atom/proc/use_check_and_message(mob/user, use_flags = 0)
	. = use_check(user, use_flags, TRUE)
```